kinto_client/cli_utils.py

```diff
@@ -37,6 +37,8 @@ def set_parser_server_options(parser=None,
                               default_auth=None,
                               default_bucket=None,
                               default_collection=None,
+                              with_bucket=True,
+                              with_collection=True,
```

Comment:
The approach taken here is worth IMO: https://github.com/mozilla-services/kinto2xml/blob/master/kinto2xml/export.py#L190-L201